crates/bevy_remote/src/builtin_methods.rs

```
@@ -793,6 +809,28 @@ pub fn process_remote_list_watching_request(
     }
 }
 
+/// Handles a `bevy/unwatch` request (unwatch a running `+watch`-suffixed request) coming from a client.
+pub fn process_unwatch_request(
+    In(params): In<Option<Value>>,
+    watching_requests: Res<RemoteWatchingRequests>,
+) -> BrpResult {
+    let watch_id = match params.map(parse::<BrpUnwatchParams>) {
+        Some(Ok(params)) => params.watch_id,
+        Some(Err(e)) => {
+            return Err(e);
+        }
+        _ => None,
+    };
+
+    for request in watching_requests.0.iter() {
+        if watch_id.is_none() || watch_id.unwrap() == request.watch_id {
+            request.message.sender.close();
+        }
+    }
+
+    Ok(Value::Null)
```

We might want to return an error if a `watch_id` is provided that doesn't exist